src/WeaverUser.coffee

```
@@ -1,26 +1,98 @@
 # Libs
-cuid = require('cuid')
-WeaverNode = require('./WeaverNode')
+cuid   = require('cuid')
+Weaver = require('./Weaver')
+loki   = require('lokijs')
 
+class WeaverUser
+  
+  constructor: () ->
+    @email
+    @username
+    @password
+    @emailVerified = false
+    
+    
+  logIn: (usr, pass) ->
+    credentials = {
+      user: usr
+      password: pass
+    }
+    coreManager = Weaver.getCoreManager()
+    users = Weaver.getUsersDB()
+    coreManager.logIn(credentials)
+    .then((res) ->
+      try
+        #
+        users.insert({user:usr,token:res.token})
+      catch error
+        console.error(error)
```

this is bad practice, throw an error or let it fail, but do not log this